script/makefiles/site.mk

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021-2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+WORKDIR = $(shell pwd)
+HUGO_VERSION = "0.101.0"
```

Comment:
Why not "latest"?

Reply:
Well, I was thinking about that and wasn't sure about the fact of using a non-fixed version that could break the compatibility at any point... Do you think it would be better to use latest here?

Reply:
This piece of code is meant for checking that website is OK when rendered by Netlify. Maybe we should check which version is used by Netlify? I mean, this should be in sync with that.